test/journeys/specs/space/global/basic.js

```diff
@@ -121,16 +121,13 @@ describe('Widget Space', () => {
     }));
 
     before('open widget for marty in browserLocal', () => {
-      const spaceWidget = '.ciscospark-space-widget';
       browserLocal.execute((localAccessToken, spaceId) => {
         const options = {
           accessToken: localAccessToken,
           spaceId
         };
         window.openSpaceWidget(options);
       }, marty.token.access_token, conversation.id);
-
-      browserLocal.waitForVisible(spaceWidget);
```

Comment:
We don't need to make sure the widget actually loads before continuing?

Reply:
@adamweeks the load is redundant as we have multiple other waits in each test. for some reason, having the waitForVisible in the before() block is what was causing the test time to explode in length

Reply:
Cool, just making sure!